src/api.v2/helpers/modelGuts.js

```diff
@@ -0,0 +1,66 @@
+const sqlClient = require('../../sql/sqlClient');
+
+// The guts of a model that uses Knexjs to store and retrieve data from a
+// database using the provided `knex` instance. Custom functionality can be
+// composed on top of this set of common guts.
+//
+// The idea is that these are the most-used types of functions that most/all
+// "models" will want to have.
+
+// Copied from https://github.com/robmclarty/knex-express-project-sample/blob/main/server/helpers/model-guts.js
+module.exports = ({
+  tableName,
+  selectableProps = [],
+  timeout = 1000,
```

Comment:
Is this in milliseconds? Is it enough time?

Reply:
Maybe we could make it longer, what number would you suggest? 3 or 4 seconds maybe?

Reply:
If we have a good model, these queries should be fast, but we're also waiting for network latency, etc. right, so maybe 4 seconds?

Reply:
ok